radicle-cli/src/commands/identity.rs

```diff
@@ -0,0 +1,519 @@
+            let meta = Metadata {
+                title: proposal.title().to_string(),
+                description: proposal
+                    .description()
+                    .unwrap_or("Enter a description")
+                    .to_string(),
+                proposed: revision.proposed.clone(),
+            };
+            let update = meta.edit()?;
+            let (_, revision) =
+                proposal.update(update.title, update.description, update.proposed, &signer)?;
+            term::success!(
+                "Identity proposal '{}' updated 🌱",
+                term::format::highlight(proposal.id)
+            );
+            term::success!(
+                "Revision '{}'",
+                term::format::highlight(revision.to_string())
+            );
+            print(&proposal, &previous, None)?;
+        }
+        Operation::List => {
+            let mut t = term::Table::new(term::table::TableOptions::default());
+            // Sort the list by the latest timestamped revisions (i.e. latest edits)
+            let mut timestamped = Vec::new();
+            let mut no_latest = Vec::new();
+            for result in proposals.all()? {
+                let (id, proposal, _) = result?;
+                match proposal.latest() {
+                    None => no_latest.push((id, proposal)),
+                    Some((_, revision)) => {
+                        timestamped.push(((revision.timestamp, id), id, proposal));
+                    }
+                }
+            }
+            timestamped
+                .sort_by(|((t1, id1), _, _), ((t2, id2), _, _)| t1.cmp(t2).then(id1.cmp(id2)));
+            for (id, proposal) in timestamped
+                .into_iter()
+                .map(|(_, id, p)| (id, p))
+                .chain(no_latest.into_iter())
+            {
+                let state = match proposal.state() {
+                    identity::State::Open => term::format::badge_primary("open"),
+                    identity::State::Closed => term::format::badge_negative("closed"),
+                    identity::State::Published => term::format::badge_positive("published"),
+                };
+                t.push([
+                    term::format::yellow(id.to_string()),
+                    term::format::italic(format!("{:?}", proposal.title())),
+                    state,
+                ]);
+            }
+            t.render();
+        }
+        Operation::Publish { id, rev } => {
+            let mut proposal = proposals.get_mut(&id)?;
+            let (rid, _) = publish_select(&proposal, rev, &previous, interactive)?;
+            let yes = confirm(interactive, "Are you sure you want to publish?");
+            if yes {
+                Proposal::publish(&proposal, &rid, signer.public_key(), &repo, &signer)?;
+                proposal.publish(&signer)?;
+                term::success!("Published new identity 🌱",);
+                print(&proposal, &previous, None)?;
+            }
```

Comment:
Does this do the merge as well? ie. once all signatures have been collected, how does one commit to the identity branch?